### DISCUSSION TITLE
VsCode persistent warnings with "@typescript-eslint/no-unsafe-assignment": "warn"

### DISCUSSION THREAD

**Discussion by renatocron** [prisma/prisma#24699] [Q&A]
### Question

Hello, Prisma Devs,
I want to ask if it's normal/expected to have those warnings on vscode/eslint, sometimes, I don't really know how to force then to happen

![image](https://github.com/prisma/prisma/assets/1919076/f07ac1f9-c336-418a-88b6-738901834227)

Even if I restart the TS Server they persist, but if I reload the Window, they go away

![image](https://github.com/prisma/prisma/assets/1919076/76ac1145-475f-46cb-9927-153864bb3cd6)

**Comment by ludralph** [comment]
Hi @renatocron 👋 

Do you have the Prisma VSCode extension installed? It's a known issue and  one user from this [issue](https://github.com/prisma/prisma/issues/14722#issuecomment-1) said,
> Opening up the definition file by pressing F12 over a referenced type will fix it so there's not need to restart. Not the best solution I know but I thought I'd mention it until a real fix comes up.

You may want to read through the issue and see if the suggestions work for you pending when we come up with an actual fix.

**Comment by renatocron** [comment]
Hello,

Yes, I do have the extension, I will try this next time, but I'll know that this would work

![image](https://github.com/prisma/prisma/assets/1919076/a3caf72f-945c-4aa1-b6b8-c8e6b251b8ef)

I have many parts of the code that references this model, but I think it's no possible to vscode know that, even for enum's where it would be easier to spot where they were imported, 
![image](https://github.com/prisma/prisma/assets/1919076/8b8bf476-0cea-46c4-9722-7b554d0fd7da)

I get this "no references found" message, but I will try anyway, thanks

**Reply by renatocron** [reply]
It's happening right now:
![image](https://github.com/prisma/prisma/assets/1919076/3bd44b26-be16-4420-9210-6432e5403357)

r.titulo says it's any, F12 send me to a few different places, but now consistently sends me to:
![image](https://github.com/prisma/prisma/assets/1919076/67b54f06-b34f-403a-a528-f4ea7f41bbb0)

but even after returning, eslint still thinks it's type any. Opening file,  again or even changing TS version does nothing
![image](https://github.com/prisma/prisma/assets/1919076/6b17073e-61df-46f6-a9ee-26edda4e16ef)

then, reload window:
![image](https://github.com/prisma/prisma/assets/1919076/32bc9e59-95bc-4aa3-b6ce-922a34671dc1)

only warning on my any

**Reply by ludralph** [reply]
Hi @renatocron 

Are you still experiencing this issue in the latest version of Prisma 6.2.1?

**Reply by renatocron** [reply]
I was not to reproduce this anymore, but I'm still on prisma v5.20.0, but since then I've upgraded to typescript 5.6.2.
I think it's safe to close this issue

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!;
